modules/kinesis/src/main/scala/com/snowplowanalytics/snowplow/enrich/kinesis/DynamoDbConfig.scala

```diff
@@ -67,9 +66,9 @@ object DynamoDbConfig {
 
   private def updateArg[F[_]: ContextShift: Sync: Timer](
     blocker: Blocker,
-    orig: EncodedOrPath,
-    getConfig: (Blocker, String, String, String) => F[Base64Json]
-  ): F[EncodedOrPath] =
+    orig: Either[Base64Hocon, Path],
```

Comment:
```suggestion
    orig: EncodedHoconOrPath,
```
Is there any particular reason why not using it?